README.md

```diff
@@ -3,3 +3,18 @@ This is the code of the GitHub Pages site containing documentation for Elastic P
 The GitHub Pages site can be found at:
 
 https://elasticpath.github.io/react-pwa-reference-storefront/
+
+
+### To Set Up Documentation Locally for Development Purposes
+*Prerequisites: Install Xcode and Command Line Tools*
+
+1. Install the latest version of rvm:
+`\curl -sSL https://get.rvm.io | bash -s stable`
+2. Install the latest version of ruby with rvm:
+`rvm install ruby --latest`
+3. Run `gem install github-pages`
```

Comment:
why are we running all these commands, what happens here? or is it like 
Run the following commands:
1.
2.
3.
?

Reply:
Yes they have to be run in order to install.  So we could do 
"Run the following commands in order" 
a) rvm....
b) rvm...
etc.